docker/README.md

```diff
@@ -0,0 +1,15 @@
+# Ubermag OOMMF container
+
+The files in this directory create the Docker image containing the OOMMF
+installation, which is used by [Ubermag](https://ubermag.github.io) if Ubermag
+uses Docker to execute oommf.
+
+This can be ignored by most users.
```

Comment:
Do we need more details on why this is can be ignored?

Reply:
I added some more detail.